DDDEastAnglia.Tests/Controllers/HomeControllerTests.cs

```diff
@@ -14,8 +13,10 @@ public void SubmitSessionLinks_ShouldBeHidden_WhenSessionSubmissionIsNotOpen()
                                         .WithSessionSubmissionClosed()
                                         .Build();
 
-            var model = new HomeController(conferenceLoader)
-                            .About().GetViewModel<AboutViewModel>();
+            var model = new HomeControllerBuilder()
```

Comment:
I'm happy to loose the builder all together if you want and replace it with a simple method in the class...